daisy/common.go

```diff
@@ -159,19 +159,36 @@ func (w *Workflow) substituteSourceVars(ctx context.Context, v reflect.Value) DE
 	})
 }
 
+type traverseAction uint
+
+const (
+	continueTraverse traverseAction = iota
+	prune
+)
+
 // traverseData traverses complex data structures and runs
 // a function, f, on its basic data types.
 // Traverses arrays, maps, slices, and public fields of structs.
 // For example, f will be run on bool, int, string, etc.
 // Slices, maps, and structs will not have f called on them, but will
 // traverse their subelements.
 // Errors returned from f will be returned by traverseDataStructure.
-func traverseData(v reflect.Value, f func(reflect.Value) DError) DError {
+// actions allows the caller to determine which action to take at a node.
+// The default action is 'continueTraverse'.
```

Comment:
It seems `continueTraverse` is a no op. What is the purpose of having it as sequential integer? Where is the value used?

Reply:
You're right, it's a no-op; the new comments describe this action as the default behavior. 

Regarding sequential integers, I used `iota` since it's my understanding of the current-best option for enums: https://github.com/golang/go/issues/28987